### DISCUSSION TITLE
Select entire tab block?

### DISCUSSION THREAD

**Discussion by DebbieAtSeam** [GitbookIO/community#565] [0. General bugs and questions] [answered]
In the GitBook GUI, how do you select an entire tab block that contains multiple tabs (in order to copy it)? The [GitBook docs](https://docs.gitbook.com/content-editor/blocks#selecting-blocks-and-interacting-with-selected-blocks) say that you can select a block by placing the cursor in the block and pressing Escape. However, that does not select the entire tab block, just the sub-block where the cursor is. I cannot seem to drag-the-cursor-to-select the tab block either. The only way that I've found to select a tab block containing multiple tabs is to drag-the-cursor-to-select some text in the paragraph before _and_ some text in the paragraph after the tab block. Then, I copy and paste it somewhere and then delete the extra copied paragraphs. This is a bit cumbersome. Is there a way to select just the tab block containing multiple tab? Thanks!

**Comment by emmerich** [comment, marked as answer]
Hey @DebbieAtSeam , here's how I usually do it:

1. Make sure your cursor is inside the content of one of the tabs
2. Press escape. You're now in block selection mode, and you should have the inner content of the tab selected
3. Press the left arrow on your keyboard, it should "jump" the selection up one level so you have the entire block selected
4. cmd+c or ctrl+c to copy, then paste wherever

Hope this helps! Let me know if you get into trouble with it.

**Reply by DebbieAtSeam** [reply]
Excellent! Many thanks, @emmerich!
